Lib/test/test_codecs.py

```diff
@@ -3191,13 +3191,16 @@ def test_incremental(self):
         self.assertEqual(decoded, ('abc', 3))
 
     def test_mbcs_alias(self):
-        # On Windows, the encoding name must be the ANSI code page
-        encoding = locale.getpreferredencoding(False)
-        self.assertTrue(encoding.startswith('cp'), encoding)
-
-        # The encodings module create a "mbcs" alias to the ANSI code page
-        codec = codecs.lookup(encoding)
-        self.assertEqual(codec.name, "mbcs")
+        # Check that looking up our 'default' codepage will return
+        # mbcs when we don't have a more specific one available
+        code_page = 9999
```

Comment:
Let's use 99999 instead, which is beyond the range of anything that Python might ever define. See [code page identifiers](https://docs.microsoft.com/en-us/windows/win32/intl/code-page-identifiers).